k8s/cluster_config/postgres_operator/values_postgres_operator.yaml

```diff
@@ -20,6 +20,6 @@ configLogicalBackup:
   # S3 server side encryption
   logical_backup_s3_sse: "AES256"
   # S3 retention time for stored backups for example "2 week" or "7 days"
-  logical_backup_s3_retention_time: ""
+  logical_backup_s3_retention_time: "1 month"
```

Comment:
Why a retention period of a single month and not another value?

Reply:
We can use any other value. Do we have some kind of backup guidelines for this?